include/rest/api_client.hpp

```diff
@@ -144,19 +243,11 @@ namespace disccord
                     }
 
                     template <typename TModel>
-                    pplx::task<void> request_multi_json(route_info& route, std::vector<TModel> body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
+                    pplx::task<void> request_multi_json(route_info& route, TModel body, const pplx::cancellation_token& token = pplx::cancellation_token::none())
```

Comment:
in that commit I added a new method that does that if you specify. it is also nicer because it abstracts all of that building of an array in the request, and does it in the model instead.